src/cluster/__tests__/fetchTopicsOffset.spec.js

```diff
@@ -38,10 +38,12 @@ describe('Cluster > fetchTopicsOffset', () => {
       createPartitioner: createModPartitioner,
     })
 
+    await producer.connect()
     await sendSampleMessages()
   })
 
   afterEach(async () => {
+    await producer.disconnect()
```

Comment:
This should use the same protection against `producer` not being assigned, to avoid noisy errors when tests fail due to problems with the cluster (see https://github.com/tulios/kafkajs/pull/795)